lib/ansible/config/routing.yml

```
@@ -1721,869 +1721,869 @@ plugin_routing:
     avi_seproperties:
-      redirect: community.general.avi_seproperties
+      redirect: community.network.avi_seproperties
     avi_serverautoscalepolicy:
-      redirect: community.general.avi_serverautoscalepolicy
+      redirect: community.network.avi_serverautoscalepolicy
     avi_serviceengine:
-      redirect: community.general.avi_serviceengine
+      redirect: community.network.avi_serviceengine
     avi_serviceenginegroup:
-      redirect: community.general.avi_serviceenginegroup
+      redirect: community.network.avi_serviceenginegroup
     avi_snmptrapprofile:
-      redirect: community.general.avi_snmptrapprofile
+      redirect: community.network.avi_snmptrapprofile
     avi_sslkeyandcertificate:
-      redirect: community.general.avi_sslkeyandcertificate
+      redirect: community.network.avi_sslkeyandcertificate
     avi_sslprofile:
-      redirect: community.general.avi_sslprofile
+      redirect: community.network.avi_sslprofile
     avi_stringgroup:
-      redirect: community.general.avi_stringgroup
+      redirect: community.network.avi_stringgroup
     avi_systemconfiguration:
-      redirect: community.general.avi_systemconfiguration
+      redirect: community.network.avi_systemconfiguration
     avi_tenant:
-      redirect: community.general.avi_tenant
+      redirect: community.network.avi_tenant
     avi_trafficcloneprofile:
-      redirect: community.general.avi_trafficcloneprofile
+      redirect: community.network.avi_trafficcloneprofile
     avi_user:
-      redirect: community.general.avi_user
+      redirect: community.network.avi_user
     avi_useraccount:
-      redirect: community.general.avi_useraccount
+      redirect: community.network.avi_useraccount
     avi_useraccountprofile:
-      redirect: community.general.avi_useraccountprofile
+      redirect: community.network.avi_useraccountprofile
     avi_virtualservice:
-      redirect: community.general.avi_virtualservice
+      redirect: community.network.avi_virtualservice
     avi_vrfcontext:
-      redirect: community.general.avi_vrfcontext
+      redirect: community.network.avi_vrfcontext
     avi_vsdatascriptset:
-      redirect: community.general.avi_vsdatascriptset
+      redirect: community.network.avi_vsdatascriptset
     avi_vsvip:
-      redirect: community.general.avi_vsvip
+      redirect: community.network.avi_vsvip
     avi_webhook:
-      redirect: community.general.avi_webhook
+      redirect: community.network.avi_webhook
     bcf_switch:
-      redirect: community.general.bcf_switch
+      redirect: community.network.bcf_switch
     bigmon_chain:
-      redirect: community.general.bigmon_chain
+      redirect: community.network.bigmon_chain
     bigmon_policy:
-      redirect: community.general.bigmon_policy
+      redirect: community.network.bigmon_policy
     checkpoint_access_layer_facts:
-      redirect: community.general.checkpoint_access_layer_facts
+      redirect: checkpoint.mgmt.access_layer_factsaccess_layer_facts
```

```suggestion
      redirect: checkpoint.mgmt.access_layer_factsaccess_layer_facts
```
incorect. and the same for the other checkpoint modules

---

and i don't see `mgmt` in the path there https://github.com/ansible-collections/checkpoint/tree/master/plugins/modules
is it ok?

---

there are two modules cp_mgmt_access_layer_facts.py and just checkpoint_access_layer_facts.

---

all the other need to be checked as well

---

Fixed, thanks